Bad dependency version react-addons-update

Looking for ^14.0.0
npm ERR! No compatible version found: react-addons-update@^14.0.0
npm ERR! Valid install targets:
npm ERR! 15.3.2, 15.3.2-rc.1, 15.3.1, 15.3.1-rc.2, 15.3.1-rc.1, 15.3.0, 15.3.0-rc.3, 15.3.0-rc.2, 15
.3.0-rc.1, 15.2.1, 15.2.0, 15.2.0-rc.2, 15.2.0-rc.1, 15.1.0, 15.1.0-alpha.1, 15.0.3-alpha.2, 15.0.3-
alpha.1, 15.0.2, 15.0.2-alpha.4, 15.0.2-alpha.3, 15.0.2-alpha.2, 15.0.2-alpha.1, 15.0.1, 15.0.0, 15.
0.0-rc.2, 15.0.0-rc.1, 0.15.0-alpha.1, 0.14.8, 0.14.7, 0.14.6, 0.14.5, 0.14.4, 0.14.3, 0.14.2, 0.14.
1, 0.14.0, 0.14.0-rc1, 0.14.0-beta3, 0.14.0-beta2, 0.14.0-beta1

npm ERR! node v4.4.6
npm ERR! npm  v3.10.7

--- a/package.json
+++ b/package.json
@@ -54,6 +54,6 @@
   "dependencies": {
     "classnames": "^2.2.5",
     "lodash": "^4.16.1",
-    "react-addons-update": "^14.0.0"
+    "react-addons-update": ">=14.0.0"
   }
 }

